script/update_dbdefs.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+set -e
+
+SCRIPT_NAME=$(basename "$0")
+
+LIST_METABRAINZ_HOSTS_DEFAULT='../docker-server-configs/scripts/list_nodes.sh'
+
+HELP=$(cat <<EOH
+Usage: $SCRIPT_NAME [<hosts list>]
+
+Copy the latest DBDefs (as stored in docker-server-configs) to all MB
+containers on the specified hosts, or all MB service hosts if none are
+specified.
+
+It's expected that your SSH config contains host aliases identical to the
+ones you've passed, e.g.:
+
+  Host aretha
+    User bitmap
+    HostName aretha.metabrainz.org
+
+The DBDefs files will only be copied into the containers if they've changed.
+Services will be sent SIGHUP where applicable.
+
+Please make sure that your working copy of docker-server-configs is
+up-to-date beforehand!
+
+Environment:
+
+  LIST_METABRAINZ_HOSTS
+    Path to the script that lists MetaBrainz hosts by service.
+    Default: $LIST_METABRAINZ_HOSTS_DEFAULT
+
+  CONTAINER_SUFFIX
+    A "musicbrainz-" container suffix to limit changes to, like "json-dump"
+    or "webservice-test".
+    Default: (none)
```

Comment:
Why is it passed through environment rather than arguments?

Reply:
It's optional, and the script already takes in a variable-length hosts list, so I didn't want to have to handle argument parsing.  In a situation like flipping `DB_READ_ONLY`, we'll be updating every container -- I think it's less common to have to update a single type of container, but it does happen sometimes.

Reply:
The main point was to allow setting multiple container suffixes at once. But then you made it work by replacing `CONTAINER_SUFFIX` with `CONTAINER_SUFFIXES`. 
To improve help message and error checking, I checked them with the list of valid container suffixes, did the same for hosts. Since these are not overlapping, there is no need for separation, so both are just arguments now.